test/test_trie_tokenizer.py

```diff
@@ -144,11 +144,11 @@ def test_trie_tokenizer(self):
 
     def test_ort_trie_tokenizer(self):
         vocab_data = util.read_file(self.vocab_file, 'rb')
-        tokr = OrtPyFunction.from_customop("TrieTokenizer", vocab=vocab_data)
+        tokr = OrtPyFunction.from_customop("TrieTokenizer", vocab=vocab_data, cpu_only=True)
         tokens = tokr(["I love you"])
         self.assertEqual(list(tokens[0]), [74, 31337, 22799])
 
-        detok = OrtPyFunction.from_customop("TrieDetokenizer", vocab=vocab_data)
+        detok = OrtPyFunction.from_customop("TrieDetokenizer", vocab=vocab_data, cpu_only=True)
```

Comment:
nit: TrieDecoder instead of TrieDetokenizer?

Reply:
decoder is too common which is also used for LLM, I intentionally uses a different name.